textattack/shared/utils/install.py

```diff
@@ -25,12 +25,13 @@ def s3_url(uri):
     return "https://textattack.s3.amazonaws.com/" + uri
 
 
-def download_if_needed(folder_name):
-    """Folder name will be saved as `.cache/textattack/[folder name]`. If it
+def download_from_s3(folder_name, skip_if_cached=True):
```

Comment:
@alexander-zap @jinyongyoo  Is the intention of `skip_if_cached` flag to control whether to force re-download?

Reply:
@alexander-zap can you answer the question from @sanchit97?

Reply:
This code was not written by me, so I sadly cannot answer this question. I think @jinyongyoo wrote this part of the code.